docs/src/main/sphinx/connector/prometheus.rst

```diff
@@ -92,6 +92,48 @@ Prometheus can be setup to require a Authorization header with every query. The
 ``prometheus.bearer.token.file`` allows for a bearer token to be read from the configured file. This file
 is optional and not required unless your Prometheus setup requires it.
 
+.. _Prometheus-type-mapping:
+
+Type mapping
+------------
+
+Because Trino and Prometheus each support types that the other does not, this
+connector :ref:`modifies some types <type-mapping-overview>` when reading or
+writing data. Data types may not map the same way in both directions between
+Trino and the data source. Refer to the following sections for type mapping in
+each direction.
+
+The connector returns fixed columns that have a defined mapping to Trino types
+according to the following table:
+
+.. list-table:: Prometheus type to Trino type mapping
+  :widths: 50, 50
+  :header-rows: 1
+
+  * - Prometheus columns
```

Comment:
Columns or types.. this is confusing

Reply:
Prometheus returns columns, and Yuya used columns in his example.

Reply:
It's because Prometheus is just a monitoring tool, and this connector exposes those metrics to Trino. So the columns are set in stone.